types/Cargo.toml

```diff
@@ -36,6 +36,7 @@ serde = { workspace = true }
 serde_bytes = { workspace = true }
 serde_json = { workspace = true }
 serde_yaml = { workspace = true }
+sha2 = { workspace = true }
```

Comment:
Is this needed? We should have already had all the deps needed for sha3256

Reply:
I used sha2 earlier for the hash. Removing this dependency as we shifted to sha3256.